frontend/app/src/components/defi/wizard/ModuleSelector.vue

```diff
@@ -1,145 +1,163 @@
 <script setup lang="ts">
-import { type Ref } from 'vue';
-import { Module, SUPPORTED_MODULES } from '@/types/modules';
+import {
+  Module,
+  SUPPORTED_MODULES,
+  type SupportedModule
+} from '@/types/modules';
 import { Section } from '@/types/status';
+import { type DataTableHeader } from '@/types/vuetify';
 
-const wasActivated = (
-  active: Module[],
-  previouslyActive: Module[],
-  module: Module
-) => active.includes(module) && !previouslyActive.includes(module);
-
-const wasDeactivated = (
-  active: Module[],
-  previouslyActive: Module[],
-  module: Module
-) => !active.includes(module) && previouslyActive.includes(module);
+const { t } = useI18n();
 
 const supportedModules = SUPPORTED_MODULES;
 const loading = ref(false);
 const search = ref('');
-const selectedModules: Ref<Module[]> = ref([]);
-const autocomplete = ref();
 
 const { activeModules } = storeToRefs(useGeneralSettingsStore());
 const { update: updateSettings } = useSettingsStore();
 
 const balancesStore = useNonFungibleBalancesStore();
 const { resetStatus } = useStatusUpdater(Section.NON_FUNGIBLE_BALANCES);
 
-const fetch = () => {
-  const callback = () => resetStatus();
-  setTimeout(callback, 800);
-};
+const headers = computed<DataTableHeader[]>(() => [
+  {
+    text: t('module_selector.table.name'),
+    value: 'name',
+    width: '100%'
+  },
+  {
+    text: t('module_selector.table.enabled'),
+    value: 'enabled'
+  }
+]);
 
-const clearNfBalances = () => {
-  const callback = () => balancesStore.$reset();
-  setTimeout(callback, 800);
-};
+const modules = computed<(SupportedModule & { enabled: boolean })[]>(() => {
+  const active = get(activeModules);
+  const filter = get(search);
+  return supportedModules
+    .filter(m =>
+      filter ? m.name.toLowerCase().includes(filter.toLowerCase()) : true
+    )
```

Comment:
Is this better, no need to do the loop if the search is empty?

`(filter ? supportedModules : supportedModules.filter(...))`